core/trino-main/src/main/java/io/trino/server/CoordinatorModule.java

```diff
@@ -460,30 +464,4 @@ private void bindLowMemoryTaskKiller(LowMemoryTaskKillerPolicy policy, Class<? e
                         .to(clazz)
                         .in(Scopes.SINGLETON)));
     }
-
-    public static class ExecutorCleanup
```

Comment:
can you explain how is your modification improving over what we had before here?

Reply:
It was just a code extraction. No improvement here. But see the last commit now.